README.md

```diff
@@ -48,22 +48,23 @@ module "example" {
 
 | Name | Version |
 |------|---------|
-| terraform | >= 0.12 |
+| terraform | ~> 0.12.0 |
+| aws | ~> 2.0 |
+| template | ~> 2.0 |
 
 ## Providers ##
 
 | Name | Version |
 |------|---------|
-| aws | n/a |
-| aws.dns | n/a |
-| template | n/a |
+| aws | ~> 2.0 |
+| aws.dns | ~> 2.0 |
+| template | ~> 2.0 |
```

Comment:
I know it is `template = "~> 2.0"` in this repo, but in [cisagov/cool-assessment-terraform](https://github.com/cisagov/cool-assessment-terraform/blob/cf132c570f5d0cbf648c3ff2c78355653fecc87f/versions.tf#L10) and [cisagov/cool-sharedservices-nessus](https://github.com/cisagov/cool-sharedservices-nessus/blob/e40dc853509a395e96b43dba7cc360a99b957cfd/versions.tf#L10) it is `2.1`. Should we standardize versions?

Reply:
Technically we should possibly be doing something like
```hcl
terraform {
  # We want to hold off on 0.13 until we have tested it.
  required_version = "~> 0.12.29"

  # If you use any other providers you should also pin them to the
  # major version currently being used.  This practice will help us
  # avoid unwelcome surprises.
  required_providers {
    aws      = "~> 2.70"
    template = "~> 2.1"
  }
}
```
since I think the `~> X.Y` syntax sets a minimum version of `X.Y` and a maximum version of "whatever is the latest release with major version `X`".  Practically I don't think it matters, since we always run `terraform init -upgrade`.  Hence `~>X.0` and `~>X.1` are indistinguishable for our purposes.

Note that [this page](https://www.terraform.io/docs/configuration/provider-requirements.html#best-practices-for-provider-versions) indicates that we should not be specifying maximum versions at all for modules that are not intended to be used as root modules.

I'm unclear what the best way forward is, but I'm fine with just bumbling ahead until we figure out what works best for us.  In the interim, I will change the template provider's version constraint to `~> 2.1`.

Reply:
See commit e0de5b5.

Reply:
Ah, right, root modules. I'm fine with however we want to forge ahead with at this time. I just remembered seeing a `template ~> 2.1` in another batch of PRs and thought I'd mention the different versions.

Reply:
I'm going to punt on this comment for now, but I do think we should revisit it after doing some more noodling.  CRITICAL PATH!